pkg/kubernetes/client.js

```diff
@@ -387,7 +387,7 @@ define([
              * have to list everything again. Still watch at the same time though.
              */
             if (lastResource) {
-                params["resourceVersion"] = lastResource;
+                params.resourceVersion = lastResource;
```

Comment:
This change seems unrelated to this pull request. Is there a reason it's included here?